### PR TITLE
Interval datatype Fix for streaming events from target to FF

### PR DESCRIPTION
**Pull request by priyanshi-yb**

### CODE REVIEW

File: yb-voyager/src/dbzm/valueConverter.go [hunk 1]
```diff
@@ -171,6 +175,12 @@ func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*s
 		if err != nil {
 			return fmt.Errorf("fetch column schema: %w", err)
 		}
+		if !checkSourceExporter(exportSourceType) && strings.Contains(strings.ToLower(colType), "interval") {
+			colType, err = conv.schemaRegistrySource.GetColumnType(strings.ToUpper(tableName), strings.ToUpper(column), conv.shouldFormatAsPerSourceDatatypes()) //assuming table name/column name is case insensitive TODO: handle this case sensitivity properly
```
Comment (amit-yb) [inline yb-voyager/src/dbzm/valueConverter.go:179]: Nit: line too long. Typically, don't go past 100 columns.

File: yb-voyager/src/dbzm/valueConverter.go [hunk 2]
```diff
@@ -171,6 +175,12 @@ func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*s
 		if err != nil {
 			return fmt.Errorf("fetch column schema: %w", err)
 		}
+		if !checkSourceExporter(exportSourceType) && strings.Contains(strings.ToLower(colType), "interval") {
```
Comment (amit-yb) [inline yb-voyager/src/dbzm/valueConverter.go:178]: Use an equality check on `colType` instead of `strings.Contains()`.

**Review by amit-yb — APPROVED** [review]
LGTM with minor nits.

File: yb-voyager/src/dbzm/valueConverter.go (hunk 2 above)
Comment (sanyamsinghal) [reply yb-voyager/src/dbzm/valueConverter.go:178]: also checkout `strings.EqualFold` if that can be useful here.
https://pkg.go.dev/strings#EqualFold

**Pull request merged**